cloudify_helm/utils.py

```diff
@@ -106,18 +105,29 @@ def get_kubeconfig_file(ctx):
 
 
 @contextmanager
-def get_values_file(ctx):
-    if ctx.node.properties.get(RESOURCE_CONFIG, {}).get('values_file'):
+def get_values_file(ctx, ignore_properties_values_file, values_file=None):
+    values_file = values_file if ignore_properties_values_file else \
+        ctx.node.properties.get(RESOURCE_CONFIG, {}).get('values_file')
+    ctx.logger.debug("values file path:{path}".format(path=values_file))
+    if values_file and not ignore_properties_values_file:
+        # It means we took values file path from resource_config
         with tempfile.NamedTemporaryFile(delete=False) as f:
             f.close()
             ctx.download_resource(
-                ctx.node.properties.get(RESOURCE_CONFIG).get('values_file'),
+                values_file,
                 target_path=f.name)
             try:
                 ctx.logger.info("using values file:{file}".format(file=f.name))
                 yield f.name
             finally:
                 os.remove(f.name)
+    elif values_file:
+        # It means we have local values file.Check if cfyuser can access it.
+        if not os.path.isfile(values_file):
+            raise NonRecoverableError(
+                "Used local values file path but Cloudify user can`t locate "
+                "it, please check file permissions.")
+        yield values_file
```

Comment:
Should we handle `else` by raising an error? Is there a situation where this outcome is valid? Also, just `yield` should be sufficient, as would `return` instead of `return None`.

Reply:
if `values_file` is local file path but we cant access it i don't want to proceed so exception raised. 
if `values file` is local  file path and i can access it so yield it.
We will enter into this else if `values_file` is local file path.

Reply:
Will fix the yield None